image/actions/build/action.yaml

```diff
@@ -0,0 +1,24 @@
+working_directory: "{{ .current_working_dir }}"
+action:
+  title: Build
+  description: Create new docker image from local files
+  options:
+    - name: tag
+      title: Custom tag
+      description: Option to preview changelog and tag
+      type: string
+      default: ""
+
+runtime:
+  type: container
+  image: image-actions-build:latest
+  build:
+    context: ./
+    args:
+      GIT_AUTHOR_NAME: ${USER}
+      GIT_AUTHOR_EMAIL: no-reply@skilld.cloud
+  command:
+    - /bin/bash
+    - /action/action.sh
+    - "{{ .tag }}"
```

Comment:
this could use common actions like https://github.com/wodby/php/blob/master/.github/workflows/workflow.yml#L48-L67